ui/src/overriden/components/Menu/RightMenuItem.js

```diff
@@ -0,0 +1,56 @@
+import { FrontSiteRoutes } from '@inveniosoftware/react-invenio-app-ils';
+import Qs from 'qs';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Dropdown, Menu, Responsive } from 'semantic-ui-react';
+
+const onClickBookRequestLink = () => {
+  const params = Qs.parse(window.location.search);
+  const queryString = params['?q'];
+  return {
+    pathname: FrontSiteRoutes.documentRequestForm,
+    state: { queryString },
+  };
+};
+
+const dropdownEntries = (
+  <Dropdown.Menu>
+    <Dropdown.Item as="a" href="https://library.web.cern.ch/resources/remote">
+      External Online <br /> Resources
+    </Dropdown.Item>
+    <Dropdown.Divider />
+    <Dropdown.Item as="a" href="#">
+      F.A.Q.
```

Comment:
not blocking, can be fixed later: maybe better the full name here, there are still people who wouldn't know what FAQ is

Reply:
😱

Reply:
Same comment above about if this is external or internal link